nano/lib/config.hpp

```diff
@@ -307,6 +309,9 @@ class network_constants
 	/** How much to delay activation of optimistic elections to avoid interfering with election scheduler */
 	std::chrono::seconds optimistic_activation_delay{ 30 };
 
+	std::chrono::milliseconds rep_crawler_normal_interval{ 1000 * 7 };
+	std::chrono::milliseconds rep_crawler_warmup_interval{ 1000 * 3 };
```

Comment:
We know that too many confirm_req are not a good thing.
If the network is in trouble and it has no sufficient weight, do we really want to flood with confirm requests?
I think we should remove this option all-together.

Reply:
I disagree. I believe the benefits of having a quick warmup phase (finding principal representatives quickly) outweigh minimally higher resource consumption. At the peak, rep crawler published ~20-25 confirm reqs per second and that phase took a fraction of a second. It then quickly throttled down.

25 requests per second pales in comparison to confirm reqs being published by active elections, which average 3k-4k per second, and they are the main bottleneck.

<img width="1436" alt="Screenshot 2024-03-05 at 14 35 10" src="https://github.com/nanocurrency/nano-node/assets/3044353/3e7db7a9-288e-4150-9b4e-45308d0da919">